src/app/apps/campaniaSalud/services/campaniaSalud.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { Server } from '@andes/shared';
+import { ICampaniaSalud } from '../interfaces/ICampaniaSalud';
+import { Observable } from 'rxjs/Rx';
+
+@Injectable()
+export class CampaniaSaludService {
+    // campaniaSeleccionada: ICampaniaSalud;
+    // campanias: ICampaniaSalud[];
+    readonly campaniaUrl = '/core/tm/campanias';
+
+    constructor(private server: Server) {
+    }
+
+    getCampanias(params?): Observable<ICampaniaSalud[]> {
```

Comment:
La palabra campañias es redundante adentro del servicio. Ver los otros metodos también.
```suggestion
    get(params?): Observable<ICampaniaSalud[]> {
```

Reply:
Resuelto en commit a0e9315de7269933dd2004550411ff72921d6c91